packages/router/src/router_module.ts

```diff
@@ -229,13 +229,14 @@ export function provideRoutes(routes: Routes): any {
  * Allowed values in an `ExtraOptions` object that configure
  * when the router performs the initial navigation operation.
  *
- * * 'enabled' (Default) The initial navigation starts before the root component is created.
- * The bootstrap is blocked until the initial navigation is complete.
+ * * 'enabled' The initial navigation starts before the root component is created.
```

Comment:
```suggestion
 * * 'enabled'- The initial navigation starts before the root component is created.
```

Nit for consistency: seems like other places are adding a dash when the property does not form part of the sentence.